AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -358,7 +359,7 @@
       "name": "[parameters('roleNameGuid')]",
       "scope": "[parameters('azureMonitorWorkspaceResourceId')]",
       "properties": {
-          "roleDefinitionId": "[concat('/subscriptions/', variables('clusterSubscriptionId'), '/providers/Microsoft.Authorization/roleDefinitions/', 'b0d8363b-8ddd-447d-831f-62ca05bff136')]",
+          "roleDefinitionId": "[concat('/subscriptions/', variables('azureMonitorWorkspaceSubscriptionId'), '/providers/Microsoft.Authorization/roleDefinitions/', 'b0d8363b-8ddd-447d-831f-62ca05bff136')]",
```

Comment:
hi Kaveesh, whats this change for ?

Reply:
This is to fix a bug in our templates. The role assingment for the Monitoring Data Reader role needs to be created in the subscription of the Azure Monitor Workspace instead of the clusters subscription as the two resources can be in different subscriptions.

Reply:
ok got it.